Library/Homebrew/language/node.rb

```diff
@@ -1,10 +1,44 @@
+require "json"
+
 module Language
   module Node
     def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache"
     end
 
-    def self.pack_for_installation
+    def self.node_system(cmd, *args)
```

Comment:
The formula object can be passed through as an argument or this module could be included inside the formula itself. That'll handle both of these cases.